### PR TITLE
DEV-3902-head-and-tail

### PR DESCRIPTION
**Pull request by zamazzal**

### CODE REVIEW

File: sh/tests/head-and-tail_test.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+echo insecure >> ~/.curlrc
+caddy start &>/dev/null
+
```
Comment (nprimo) [inline sh/tests/head-and-tail_test.sh:10]: I would check if the file has been submitted correctly as it has been done in the exercise `to-git-or-not-to-git` (https://github.com/01-edu/public/blob/master/sh/tests/to-git-or-not-to-git_test.sh)

File: sh/tests/head-and-tail_test.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
```
Comment (nprimo) [inline sh/tests/head-and-tail_test.sh:2]: should we add a check to ensure the student solution is not a simple `echo`?

Maybe something like
```bash
if [[ $(cat <path-to-student-solution> | grep echo | wc -l) -ne 0 ]];
then
  echo "Bad solution";
fi;
```
Or, viceversa, check that the solution is using `curl`, `head` and `tail` keywords
Or both checks :)

File: subjects/devops/head-and-tail/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```console
+$tail file
+<last 10 lines>
+$tail -1 file
+<last 1 line>
+$
+```
+
+May you will need to use `pipe (|)` and `&&`.
```
Comment (nprimo) [inline subjects/devops/head-and-tail/README.md:50]: **SUGGESTION** I will rephrase this as follows:
You will need to use pipes (`|`) and the logical operator `&&`

**Review by nprimo — CHANGES_REQUESTED** [review]

File: sh/tests/head-and-tail_test.sh (hunk 1 above)
Comment (zamazzal) [reply sh/tests/head-and-tail_test.sh:10]: fixed

File: sh/tests/head-and-tail_test.sh (hunk 2 above)
Comment (zamazzal) [reply sh/tests/head-and-tail_test.sh:2]: fixed

File: subjects/devops/head-and-tail/README.md (hunk 3 above)
Comment (zamazzal) [reply subjects/devops/head-and-tail/README.md:50]: they are different solutions to solve this exercise, and it's just a hint

**Review by nprimo — COMMENTED** [review]
Isn't the `README.md` supposed to be in the main repo `public/subjects`?
